client/src/pages/ChefContent.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import { Edit } from "@material-ui/icons";
```

Comment:
You're importing from icons but it isn't installed in the package.json, can you npm install that so it's easier for others to use locally?

Reply:
I have it installed locally actually as it wouldn't compile without it, dont know why its not showing up in the dependencies